frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+import formatCurrency from '../../lib/format_currency';
+import filterCollection from '../../lib/filter_collection';
 
 export default class PublicGroupHero extends React.Component {
```

Comment:
would be better to have this as functional component `export default ({group}) => {...}`